spec/models/ability_spec.rb

```diff
@@ -66,152 +66,142 @@
     SolrDocument.new(solr_rsp['response']['docs'].first)
   end
 
-  after do
+  after(:all) do
     cleanup_jetty
   end
 
   describe "a user with no roles" do
-    let(:guest) { nil }
-    subject { Ability.new(guest) }
-
-    it { is_expected.not_to be_able_to(:create, GenericFile) }
-    it { is_expected.not_to be_able_to(:edit, file) }
```

Comment:
After this refactor this spec went from ~90 seconds to <20 seconds. Why? Because we are using the one line `it {}` syntax throughout this file. This should only ever be used when your code is in isolation. If it is not (ones that integrate with a DB, an external webservice, or end-to-end-tests). Then you should not use this. I would argue you shouldn't use it anways. Having it in a it block is much more expressive and easier to understand. The big issue here is the before/after blocks are ran on every one of these and there was like 80+ in this spec which makes this super slow.